.github/workflows/test.yml

```diff
@@ -27,4 +27,6 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
       - run: npm install
+      - run: cd test/fixtures/sample && npm install
+      - run: cd test/fixtures/sample && npm run build
```

Comment:
even though the beforeAll in test/index.js is supposed to run the build (this works locally), this was required for tests to pass in github actions